Rawl_B502/genelach_clainne_énna.trig

```diff
@@ -29,7 +29,7 @@
     a foaf:Person;
     irishRel:genName "Libir";
     irishRel:nomName "Liber";
-    rel:childOf <#Libir-e2ba2c27>.
+    rel:childOf <#Cóelán>.
```

Comment:
I agree but should he not now be `owl:sameAs <#Libir-e2ba2c27>`?

Reply:
I am not sure I follow:

> Feradach et Colmán dá mc Áeda m. Libir et Cóelán m. Libir m. Dallíni m. Énnae m. Láegaire m. Néill

I believe there are two different Libir's here.  Cóelán is the son of one and has some kind of relationship with the other.  So maybe the below instead?

```turtle
<#Áeda>
    a foaf:Person;
    irishRel:nomName "Áeda";
    rel:childOf <#Libir>, <#Cóelán>.

<#Libir>
    a foaf:Person;
    irishRel:genName "Libir";
    irishRel:nomName "Liber";
    rel:childOf <#Cóelán>.

<#Cóelán>
    a foaf:Person ;
    irishRel:genName "Cóelán" ;
    rel:childOf <#Libir-e2ba2c27>.

<#Libir-e2ba2c27>
    a foaf:Person;
    irishRel:genName "Libir";
    irishRel:nomName "Liber";
    rel:childOf <#Dallíni>;
    owl:sameAs <http://example.com/LL/loegaire.trig#Libuir>.
```

Reply:
This is another instance of very confusing wording on the source's part. However, I think both instances of "Libir" refer to the same person. Later on, in Rawl_B502/genelach_ceníuil_loígaire.trig, we find the following:

> ... Máel Dúin Dergainich m. Colmáin m. Áeda m. Libir m. Dallíni m. Énnai m. Lóegaire m. Néill Noígiallaig.

So Liber, father of Áed, and Liber, son of Daillíne, are considered to be the same person here and this isn't incompatible with the text in Rawl_B502/genelach_clainne_énna.trig .

Reply:
OK, I have amended to:

```turtle
<#Áeda>
    a foaf:Person;
    irishRel:nomName "Áeda";
    rel:childOf <#Libir>, <#Cóelán>.

<#Libir>
    a foaf:Person;
    irishRel:genName "Libir";
    irishRel:nomName "Liber" .        

<#Cóelán>
    a foaf:Person ;
    irishRel:genName "Cóelán" ;
    rel:childOf <#Libir-e2ba2c27>.

<#Libir-e2ba2c27>
    a foaf:Person;
    irishRel:genName "Libir";
    irishRel:nomName "Liber";
    rel:childOf <#Dallíni>;
    owl:sameAs <http://example.com/LL/loegaire.trig#Libuir>,
               <#Libir>.
```